accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx

```diff
@@ -553,8 +556,7 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
                       >
                         <li>
                           <p style={{ marginTop: "0rem" }}>
-                            Highlight the selected element, or the element
-                            containing the selected issue, in the browser's
+                            Highlight the selected element (purple highlight), in the DOM under the browser's
```

Comment:
This sentence is the first of two things that happen when you select an element or issue.  It is referring to the highlighting of the element in the DOM view, which is not a purple highlight.